### ISSUE TITLE
Не удается инициализировать расширение

### ISSUE THREAD

**Issue by ScyllaSYS** [tyan/BCFierForRenga#28]
Проблема с установкой плагина. Разархивировал в папку plugins, выдало ошибку при открытии Ренги.
![Снимок экрана 2024-11-26 172907](https://github.com/user-attachments/assets/43037642-8197-4c01-932d-d7041b659e53)

**Comment by asyatyan** [comment]
Скорее всего, ваша Windows заблокировала содержимое скачанного из интернета файла.
После скачивания архива нужно открыть Свойства архива и поставить галочку Разблокировать, после этого распаковать архив и разместить его в папке Plugins.

**Comment by ScyllaSYS** [comment]
> Скорее всего, ваша Windows заблокировала содержимое файла, скачанного из интернета. После скачивания архива нужно открыть «Свойства архива» и поставить галочку «Разблокировать», после чего распаковать архив и поместить его в папку «Plugins».

К сожалению, пробовал этот способ, не помогло

**Comment by tyan** [comment]
Подскажите на какой версии Renga запускаете. Также проверьте, что в системе установлен .Net 4.8.

**Comment by ScyllaSYS** [comment]
> Подскажите на какой версии Renga запускаете. Также проверьте, что в системе установлен .Net 4.8.

Версия Renga 8.3, .Net 4.8 установлен

**Comment by tyan** [comment]
- Пришлите скрин состава папки плагина и путь по которому положили плагин
- Работают ли у вас другие плагины, скаченные с гитхаб, например ModelExplorer?

**Comment by ScyllaSYS** [comment]
![image](https://github.com/user-attachments/assets/2658afc8-fb77-4c07-9682-da15a9ceb16d)
Путь: C:\Program Files\Renga Professional\plugins
Плагин ModelExplorer работает

**Comment by tyan** [comment]
А полный путь до папки плагина как выглядит? С учетом имени папки?

**Comment by ScyllaSYS** [comment]
![image](https://github.com/user-attachments/assets/42e6599c-a31a-42a2-b1df-3d517e6b6a65)

**Comment by tyan** [comment]
Пока хороших идей нет.

Проверьте еще раз, вы включали эту галочку и нажимали "Применить"?

![image](https://github.com/user-attachments/assets/ef37dad9-530d-4637-863f-06c36a6a40de)

Скажите, вы используете еще какие-то плагины параллельно (кроме BCFier и ModelExplorer)?

**Comment by asyatyan** [comment]
Вот здесь можно посмотреть, какое ещё расширение не инициализировалось
%LocalAppData%\Renga Software\Renga Professional\AecApp.log

**Comment by tyan** [comment]
Попробуйте отключить все плагины кроме BCFier, может конфликт какой-то по библиотекам с каким-то плагином. Посмотрите в файл %USERPROFILE%\AppData\Local\Renga Software\Renga Professional\AecApp.log, что там написано.

**Comment by ScyllaSYS** [comment]
Да, конфликт с плагином САПР-Альфа. Ошибок этого плагина не было до того, пока BCFier не подцепился 
[AecApp.log](https://github.com/user-attachments/files/17930268/AecApp.log)

**Comment by tyan** [comment]
Понял, посмотрим.

**Comment by tyan** [comment]
А проверьте пожалуйста САПР-Альфа самого по себе, без всех остальных плагинов

**Comment by ScyllaSYS** [comment]
![image](https://github.com/user-attachments/assets/8a79ab25-9169-4a9d-9ca1-c068f195bf61)
Отдельно работает превосходно, без нареканий. Версия 1.14

**Comment by tyan** [comment]
Проблема на стороне Renga. Поправят в ближайшем релизе.
